CONTRIBUTING.md

```diff
@@ -274,6 +493,13 @@ Founders
 - Chris Higgs (@chiggs)
 - Stuart Hodgson (@stuarthodgson)
 
+### Getting in Contact with a Maintainer
+
+Cocotb's maintainers are always open to direct inquiries.
```

Comment:
Not sure about this section, we may be advertising more than we can fulfill.
It may also be a good idea to mention that we are "unpaid volunteers" or at least not paid support staff.